themes/jaeger-docs/layouts/partials/docs/header.html

```diff
@@ -67,10 +68,21 @@
   <hr class="hr" />
 
   {{ if .Params.children }}
+    {{ $relPermalink := .RelPermalink -}}
     <p>See also:</p>
     <ul>
       {{ range .Params.children }}
       {{ $url := printf "/docs/%s/%s/" $version .url }}
+      {{ with $page.GetPage $url -}}
+        {{ $url = .RelPermalink -}}
+      {{ else -}}
+        {{ $url = (printf "%s%s" $relPermalink .url) -}}
```

Comment:
Can this not reference the field directly, without variable?

Reply:
You're referring to `$relPermalink`?
You'd like to see `.RelPermalink`?

Reply:
If so, we're inside the `range`, so `.` doesn't refer to the page anymore. Hence the assignment to `$relPermalink` outside of the scope of `range`.